zerver/lib/outgoing_webhook.py

```diff
@@ -317,10 +318,12 @@ def do_rest_call(
 ) -> Optional[Response]:
     """Returns response of call if no exception occurs."""
     try:
+        start_time = perf_counter()
         response = service_handler.make_request(
             base_url,
             event,
         )
+        logging.info("Made outgoing webhook request in %f seconds", perf_counter() - start_time)
```

Comment:
Hmm, I guess we don't have the `user_id` for the bot; if we did, I'd be happy to have a permanent logging statement for this; I can imagine it being useful for debugging to just have records of all outgoing webhook events.

Reply:
We do, actually.  Repushed with that, and removed the "temporary" bits from the commit message.